ValidateProfileEmail/handler.ts

```diff
@@ -265,7 +246,8 @@ export const ValidateProfileEmail = (
   validationCallbackUrl: ValidUrl,
   timeStampGenerator: () => number,
   profileEmails: IProfileEmailReader,
-  FF_UNIQUE_EMAIL_ENFORCEMENT_ENABLED: (fiscalCode: FiscalCode) => boolean
+  FF_UNIQUE_EMAIL_ENFORCEMENT_ENABLED: (fiscalCode: FiscalCode) => boolean,
+  confirmChoiceUrl: ValidUrl
```

Comment:
Could be more readable regroup all the validation urls under the same object property. For example
```typescript
emailValidationUrls: {
    validationCallbackUrl: ValidUrl,
    confirmValidationUrl: ValidUrl
}
```

Reply:
done in https://github.com/pagopa/io-functions-public/pull/72/commits/7d115d231be6393d3b76e2db3ca557f7419a4eab